src/Library.js

```diff
@@ -3,5 +3,6 @@
 module.exports = {
   // createLibrary,
   // addBook,
-  // checkoutBook
+  // checkoutBook,
+  // takeStock
```

Comment:
I might update the file name to be a lower-case library for this file and the test file!